spark_api.gemspec

```diff
@@ -28,33 +28,26 @@ Gem::Specification.new do |s|
   s.files              = Dir["{History.txt,LICENSE,Rakefile,README.md,VERSION}", "{bin,lib,script}/**/*"]
   s.test_files         = Dir["spec/{fixtures,unit}/**/*", "spec/*.rb"]
   s.executables        = ["spark_api"]
-  s.default_executable = %q{spark_api}
   s.require_paths      = ["lib"]
   
-  s.add_dependency 'faraday', '~> 0.9.0'
+  s.add_dependency 'faraday', '>= 0.16.0'
```

Comment:
There are a lot of versions between 0.9 and what's now allowed. No impacts to defaults like timeouts or anything else that might negatively impact things downstream?

Reply:
Reading through the version history, nothing jumps out

Reply:
If I recall correctly, the upgrade to 0.9.0 from before that had a TON of interface and middleware consequences, so this is a tad surprising to me. Unfortunately this seems to have been prior to our gem rebranding 8+ year ago, so I can't find a SHA to reference very easily.

That said, it seems normal for a project to have more of those growing pains in the earlier versions.